### PR TITLE
only trace valid requests

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
Realized while looking at the traces that we had bunch of traces for weird URLs.
Turns out is people trying to find security holes in our apps.
We do not want to send those all traces to our zipkin. This PR solves the problem for Rails apps.

@jamescway @jfeltesse-mdsol

### CODE REVIEW

File: lib/zipkin-tracer/rack/zipkin-tracer.rb [hunk 1]
```diff
@@ -59,6 +59,16 @@ def call(env)
     end
 
     private
+
+    # If the request is not valid for this service, we do not what to trace it.
+    def valid_request?(env)
```
Comment (jfeltesse-mdsol) [inline lib/zipkin-tracer/rack/zipkin-tracer.rb]: Can the naming be improved? `valid_` is too broad IMHO. `processable_` or something?

Comment (jcarres-mdsol) [inline lib/zipkin-tracer/rack/zipkin-tracer.rb]: routable?

Comment (jfeltesse-mdsol) [inline lib/zipkin-tracer/rack/zipkin-tracer.rb]: `acceptable_`, `routable_`, `processable_`, `something completely different`, pick one :wink:

**Comment by jamescway** [comment]
:+1:

**Comment by jamescway** [comment]
released 0.6.2

**Pull request merged**